templates/org/home.tmpl

```diff
@@ -28,6 +28,9 @@
 			<div class="ui eleven wide column">
 				{{if .CanCreateOrgRepo}}
 					<div class="text right">
+            {{if not .DisabledMirrors}}
+						  <a class="ui green button" href="{{AppSubUrl}}/repo/migrate?org={{.Org.ID}}&mirror=1">{{.i18n.Tr "new_mirror"}}</a>
```

Comment:
Org/Mirror can become nil, I think you need to conditionally add those params only when the are not nil to not break regular migrations.

Reply:
Sorry, but how can `.Org` be nil? This would be on the homepage of an Org so wouldn't `.Org` have to exist? Existing migrations from the dropdown to the top right will not be sending org and mirror params. In that case, from the repo selection page (second screenshot), `org=&mirror=` will be sent, which will have no impact on the form in screenshot 3.

Reply:
Ah, probably fine. I misinterpreted the `serviceType == 0` part in the code. If that template only renders in that `if` clause it cannot become nil under regular means.